src/editor-theme-fluent.js

```diff
@@ -13,7 +13,7 @@ function theme(acequire, exports, module) {
 
     exports.cssText = `
         .ace-fluent {
-            background-color: #f7f7f7;
+            background-color: #e8e8e8;
```

Comment:
Please don't change this here but rather override the style in a CSS class specific to projectfluent.io. ` src/editor-theme-fluent.js` is shared between projectfluent.io and the Playground.